README.md

```diff
@@ -247,7 +247,7 @@ Focus is on the specific task.
 *Plugins and libraries for implementing authentication and authorization.*
 
 - [Authenticate plugin](https://github.com/FriendsOfCake/Authenticate) - Authentication classes for AuthComponent.
-- [CakeDC Users plugin](https://github.com/CakeDC/users/tree/3.x) - For allowing users to register and login, manage their profile and account validation. Social Login, RBAC, remember me, SuperUser authorize, forgot password, admin management, etc.
+- [CakeDC Users plugin](https://github.com/CakeDC/users/tree/3.1.x) - For allowing users to register and login, manage their profile and account validation. Social Login with Facebook and Twitter, RBAC, Remember me, SuperUser authorize, forgot password, admin management, etc.
```

Comment:
@steinkel, maybe shorten the description - getting too long compared to all the rest.

My suggestion:

```
Complete user management (admin panel, remember me, etc), Facebook/Twitter login, RBAC and more
```